yarn.lock

```diff
@@ -505,6 +525,15 @@
     buffer-from "^1.1.1"
     snappy "^6.3.5"
 
+"@chainsafe/ssz@^0.8.20":
```

Comment:
Warning, this should not be in the yarn.lock. Who is depending in the old versions?